src/LinearMap/Index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { indexLine, indexTick, indexTickLabel } from "../style";
+import { useBuilder } from "./hooks/useBuilder";
+import { Tick } from "./Tick";
+
+
+/**
+ * The Index component renders the Linear Map's:
+ * 		1. name (center or bottom)
+ * 		2. number of bps (center or bottom)
+ * 		3. index ticks and numbers along the Linear Map
+ */
+
+export interface IndexProps {
+  height: number
+  width: number
+  seqLength: number
+}
+
+export function Index(props: IndexProps) {
```

Comment:
Can I use functional components @jjti ou should I maintain the class components being used throughout the project?